pages/components.css

```diff
@@ -58,5 +58,10 @@
 @import url(../blocks/input/__wrapper/_type/input__wrapper_type_phone.css);
 @import url(../blocks/input/__wrapper/_type/input__wrapper_type_email.css);
 
+/* text-input UI Component*/
```

Comment:
@MarPostovik This comment' formatting a bit off comparing to other comments

Reply:
Changed ✅